src/transformers/models/parakeet/configuration_parakeet.py

```diff
@@ -150,6 +150,65 @@ def __init__(
         self.initializer_range = initializer_range
 
 
+
+class ParakeetTDTDecoderConfig(PreTrainedConfig):
+    model_type = "parakeet_tdt_decoder"
+    keys_to_ignore_at_inference = ["past_key_values"]
+    output_hidden_states = False
```

Comment:
It already defaults to False ([here](https://github.com/huggingface/transformers/blob/a8f32a0e9c2fec0f196fab5d8316a03f35d0c528/src/transformers/configuration_utils.py#L204)) so no need to set.